.goreleaser.yml

```diff
@@ -7,7 +7,7 @@ builds:
     env:
       - CGO_ENABLED=0
     ldflags:
-      - "-s -w -X 'github.com/conduitio/conduit-connector-connectorname.version={{ .Tag }}'"
+      - "-s -w -X 'github.com/github.com/conduitio-labs/conduit-connector-weather.version={{ .Tag }}'"
 checksum:
   name_template: checksums.txt
 archives:
```

Comment:
We need to use the name template instead of replacements (see changes done in https://github.com/ConduitIO/conduit-connector-template/pull/10).